samples/kmeans-mnist-processingjob.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: sagemaker.aws.amazon.com/v1
+kind: ProcessingJob
+metadata:
+  name: kmeans-mnist-processing
+spec:
+  environment:
+    - name: MYVAR
```

Comment:
mostly yes, but not always. Tag field is an example
I see these samples as human readable way to enable customer write their spec rather than figuring out each input from CRD and since environment is a good feature for processing I have added it

Reply:
Plus these values are _valid_, they're just not useful.